src/app/components/media/Media.js

```diff
@@ -73,7 +73,6 @@ const MediaContainer = Relay.createContainer(MediaParentComponent, {
           id
           dbid
           locked
-          permissions
```

Comment:
You don't need permissions anymore?

Reply:
I'm getting the permission value from the ProjectMedia object. Since we don't have different perm values for each annotation I think it's more economic to get it from the parent obj